build.ps1

```diff
@@ -256,12 +256,23 @@ function cleanExtension([string] $bitness) {
     Remove-Item -Recurse -Force "$privateSiteExtensionPath\$bitness\runtimes\linux" -ErrorAction SilentlyContinue
     Remove-Item -Recurse -Force "$privateSiteExtensionPath\$bitness\runtimes\osx" -ErrorAction SilentlyContinue
 
-    Get-ChildItem "$privateSiteExtensionPath\$bitness\workers\node\grpc\src\node\extension_binary" | 
+    Get-ChildItem "$privateSiteExtensionPath\$bitness\workers\node\grpc\src\node\extension_binary" -ErrorAction SilentlyContinue | 
     Foreach-Object {
         if (-Not ($_.FullName -Match "win32")) {
             Remove-Item -Recurse -Force $_.FullName
         }
     }
+
+    if ($bitness -eq "64bit") {
+        $keepRuntimes = @('win', 'win-x64', 'win10-x64')
+    } elseif ($bitness -eq "32bit") {
+        $keepRuntimes = @('win', 'win-x86', 'win10-x86')
+    } else {
+        # if no bitness is specified, grab all Windows 10 folders
+        $keepRuntimes = @('win', 'win-*', 'win10-*')
```

Comment:
@pragnagopa @fabiocav @alrod - this logic I'm unsure of because I don't know what the `.no-runtime` folder is for

Reply:
`.no-runtime` means the runtime is not generated for specific platform (x86 or x64).

Reply:
I take it we don't want any of the ARM folders though, correct?

Reply:
I think so. Anyway you should test the private site extension with both platforms (x86 and x64).

Reply:
That's correct. None of the ARM runtime specific dependencies are applicable here.